src/main/java/uk/gov/justice/hmpps/prison/service/IncidentService.java

```diff
@@ -11,19 +11,16 @@
 public class IncidentService {
 
     private final IncidentCaseRepository repository;
-    private final BookingService bookingService;
 
-    public IncidentService(final IncidentCaseRepository repository, final BookingService bookingService) {
+    public IncidentService(final IncidentCaseRepository repository) {
         this.repository = repository;
-        this.bookingService = bookingService;
     }
 
     public IncidentCase getIncidentCase(@NotNull final long incidentCaseId) {
         return repository.getIncidentCases(List.of(incidentCaseId)).stream().findFirst().orElseThrow(EntityNotFoundException.withId(incidentCaseId));
     }
 
     public List<IncidentCase> getIncidentCasesByOffenderNo(@NotNull final String offenderNo, final List<String> incidentTypes, final List<String> participationRoles) {
-        bookingService.getOffenderIdentifiers(offenderNo, "SYSTEM_USER");
```

Comment:
Preauthorize - with SYSTEM_USER and VIEW_INCIDENTS on controller satisfies access, so this is not required here (only offender-risk-profiler calls this which is a client-creds only call)